scalableminds/webknossos
# Change shading of meshes

This PR changes the way loaded meshes are rendered from rendering the normals to lambert-shading which is used for the isosurface-meshes. 

### URL of deployed dev instance (used for testing):
- <https://changemeshshading.webknossos.xyz>

### Steps to test:
- Open a hybrid tracing.
- Upload a mesh using the meshes tab
- The mesh should now have a color and the normal should no longer be rendered. The way the loaded mesh is rendered should be the exact same way isosurfaces are rendered.

### Issues:
- fixes #5011

------
